src/SCTPWrapper.cpp

```diff
@@ -98,6 +98,42 @@ void SCTPWrapper::OnNotification(union sctp_notification *notify, size_t len) {
       break;
     case SCTP_STREAM_RESET_EVENT:
       SPDLOG_TRACE(logger, "OnNotification(type=SCTP_STREAM_RESET_EVENT)");
+      struct sctp_stream_reset_event reset_event;
+      reset_event = notify->sn_strreset_event;
+      for (int i = 1; i < 2; i++) {
```

Comment:
Lets drop the `for` here - we're only closing one data channel at a time.

Alternatively, if the `sctp_stream_reset_event` can actually hold multiple streams to reset, lets loop across all of them.